components/org.wso2.carbon.identity.sso.saml.common/src/test/java/org/wso2/carbon/identity/sso/saml/common/UtilTest.java

```diff
@@ -0,0 +1,129 @@
+package org.wso2.carbon.identity.sso.saml.common;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.mockito.Mock;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+import org.wso2.carbon.identity.sso.saml.stub.types.SAMLSSOServiceProviderDTO;
+
+import java.lang.reflect.Array;
+import java.net.URI;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.powermock.api.mockito.PowerMockito.when;
+import static org.testng.Assert.*;
+
+
+public class UtilTest {
+
+    private static int singleLogoutRetryCount = 5;
+    private static long singleLogoutRetryInterval = 60000;
+
+
+    @Test
+    public void testGetSingleLogoutRetryCount() throws Exception {
+        int singleLogoutRetryC= Util.getSingleLogoutRetryCount();
+        Assert.assertEquals(singleLogoutRetryC,singleLogoutRetryCount);
+    }
+
+    @Test
+    public void testSetSingleLogoutRetryCount() throws Exception {
+        Util.setSingleLogoutRetryCount(6);
+        Assert.assertEquals(Util.getSingleLogoutRetryCount(),6);
+        Util.setSingleLogoutRetryCount(singleLogoutRetryCount);
+    }
+
+    @Test
+    public void testGetSingleLogoutRetryInterval() throws Exception {
+        long singleLogoutRetryInt =Util.getSingleLogoutRetryInterval();
+        Assert.assertEquals(singleLogoutRetryInt,singleLogoutRetryInterval);
+    }
+
+    @Test
+    public void testSetSingleLogoutRetryInterval() throws Exception {
+        Util.setSingleLogoutRetryInterval(70000);
+        Assert.assertEquals(Util.getSingleLogoutRetryInterval(),70000);
+        Util.setSingleLogoutRetryInterval(singleLogoutRetryInterval);
+    }
+
+    @DataProvider(name = "provideHttpStatusCode")
+    public Object[][] createData1() {
+        return new Object[][] {
+                { 200, true },
+                { 300, false},
+                { 100, false},
+                { 0, false},
```

Comment:
u also need values in the range 4xx and 5xx. And some other values within 2xx range